framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -780,6 +780,19 @@ static EventStrings events_to_string[] = {
     { XR_TYPE_EVENT_DATA_USER_PRESENCE_CHANGED_EXT, "XR_TYPE_EVENT_DATA_USER_PRESENCE_CHANGED_EXT" },
 };
 
+bool IsHandledEventType(XrStructureType type)
+{
+    size_t len = sizeof(events_to_string) / sizeof(EventStrings);
+    for (size_t ii = 0; ii < len; ++ii)
+    {
+        if (type == events_to_string[ii].type)
+        {
+            return true;
+        }
+    }
+    return false;
+}
+
```

Comment:
On the "default return" wondering if we should labled it uniquely from XR_TYPE_UNKNOWN, as (ironically) TYPE_UNKNOWN is a known enum.  That an event has XR_TYPE_UNKNOWN (0) is a distinct reporting state from type not found in list of known type enum values. "unknown extension event" ?

Reply:
The reason I didn't is in case we end up intentionally ignoring known events for some reason.  But I could consider it at some point in the future.